feat: support global keyframes (#27)
wixplosives/stylable.io: docs/references/keyframes.md

</div>
```

## Global keyframes

> To create global (unscoped) keyframes, **Stylable** provides a function that takes the global keyframes name as a parameter.

Reviewer: ```suggestion
To create global (unscoped) keyframes, **Stylable** provides the `stGlobal` function which accepts a global keyframe name as its parameter.
```

Reviewer 2: not sure about the `*the* stGlobal`

Author: I can agree with @barak007 on this one.
